setup/setup_validation_docker.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash
+
+#######################################################################################
+#  This scripts is doing basic docket on Centos 7 only  #
+#######################################################################################
+
+set +e
+
+#Postgresql configuration file
+PG_FILE=/var/lib/pgsql/9.*/data/postgresql.conf
+
+function showUsage {
+    echo "Usage: " `basename "$0"` "[options]"
+    echo -e "\t[--validate]          Validate server configuration"
+    echo -e "\t[--install]           Install docker and docker-compose"
+    echo -e "\t[-h,--help,--usage]   Display this help and exit."
+    echo -e "\t[-v,--verbose]        Print commands before executing them."
+}
+
+
+# Validates
+function validate {
+    sudo yum install bc -y > /dev/null
+
+    echo -e "\nShotgun Enterprise Setup Validation\n"
+    echo -e "\nHIGH LEVEL"
+    echo "=========="
+
+    # Hostname
+    printf "Hostname               : %-10s \n" $HOSTNAME
+
+    # Os release number checker
+    MOSR=`rpm -q --qf "%{VERSION}" $(rpm -q --whatprovides redhat-release)`
+
+    if (( $(echo "$MOSR > 6" |bc -l) )); then
+	export OVER="OK"
+    else
+	export OVER="BAD"
+    fi
+
+    # Os release checker
+    if [ $(cat /etc/*release | grep -c CentOS) -ne 0 ]; then
+	export DISTRO='centos'
+	export OVER="OK"
+    elif [ $(cat /etc/*release | grep -c 'Red Hat') -ne 0 ]; then
+	export DISTRO='redhat'
+	export OVER="OK"
+    else
+	export DISTRO="unknown"
+	export OVER="BAD"
+    fi
+    printf "RHEL 7/CentOS 7        : %-10s %-10s %-10d\n" $OVER $DISTRO $MOSR
+
+    #Kernel version checker
+    DRV=3.10
+    CCV=`uname -r|awk -F. '{print $1"."$2}'`
+
+    if (( $(echo "$CCV >= $DRV" |bc -l) )); then
+	export OVER="OK"
+    else
+	export OVER="BAD"
+    fi
+    printf "Kernel version         : %-10s %-10s\n" $OVER $CCV
+
+    # CPU model
+    CPU=`cat /proc/cpuinfo |grep "model name" |awk -F: '{print $2}'| uniq`
+    printf "CPU model              :$CPU\n"
+
+    # Number of CPU
+    NCPU=`cat /proc/cpuinfo |grep "model name" |awk -F: '{print $2}'| wc -l`
+    if (( $NCPU < 6)); then
+	export OVER="BAD"
+    else
+	export OVER="OK"
+    fi
+    printf "NB CPU                 : %-10s %-10d\n" $OVER $NCPU
+
+    # Memory check
+    MEM=`cat /proc/meminfo |grep "MemTotal:"|awk '{print $2}'`
+    TMEM=`echo "scale = 3; $MEM/1024/1024" |bc`
+
+    if (( $MEM < 67108864 ));then
+	export OVER="BAD"
+    else
+        export OVER="OK"
+    fi
+    printf "RAM                    : %-10s %-10sKB\n" $OVER $MEM
+
+    # SELinux check
+    export OVER="OK"
+    if [ ! \( `getenforce` == "Permissive" -o `getenforce` == "Disabled" \) ]; then
+      export OVER="BAD"
+    fi
+    printf "SE Linux               : %-10s %-10s\n" $OVER `getenforce`
```

Comment:
I just don't know if Docker and our containers will work or not.  Let's play safe and require SELinux to be disabled on host like before